main.cpp

```diff
@@ -133,31 +192,40 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 int main()
 {
-    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
-    //1)
-    
-    //2)
-    
-    //3)
-    
-    //4)
-    
-    //5)
-    
-    //6)
-    
-    //7)
-    
-    //8)
-    
-    //9)
-    
-    //10)
-    
+    //1)
+    auto mathsResult = doMaths(3.13);
+
+    //2)
+    auto fullInitials = concatenateInitials('A', 'B', 'C');
+
+    //3)
+    //auto viewProject = setupProjectView();
```

Comment:
You still need to call this function. 
If it doesn't return anything, don't write `auto variableName = ` on the left side of the function call.
- [ ] revise